src/status_im/ui2/screens/chat/messages/message.cljs

```diff
@@ -128,13 +128,14 @@
           (:parsed-text content)))
 
 (defn quoted-message
-  [{:keys [message-id chat-id]} reply pin?]
-  (let [{:keys [deleted? deleted-for-me?]} (get @(re-frame/subscribe [:chats/chat-messages chat-id])
-                                                message-id)
-        reply                              (assoc reply
-                                                  :deleted?        deleted?
-                                                  :deleted-for-me? deleted-for-me?
-                                                  :chat-id         chat-id)]
+  [{:keys [message-id chat-id]} _ pin?]
+  (let [{:keys [deleted? deleted-for-me?]
+         :as   quoted-message} (get @(re-frame/subscribe [:chats/chat-messages chat-id])
+                                    message-id)
+        reply                  (assoc quoted-message
+                                      :deleted?        deleted?
+                                      :deleted-for-me? deleted-for-me?
+                                      :chat-id         chat-id)]
```

Comment:
- redundant `assoc` for keys `:deleted?` and `:deleted?-for-me`
- Not sure, just curious, is quoted-message has chat-id?

Reply:
just checked, message has chat-id, whole assoc is unnecessary 
![image](https://user-images.githubusercontent.com/17097240/221588488-ba4d9ee9-ce99-45ec-9bfa-b41d6fa5180d.png)

Reply:
Removed, Thanks Parvesh